src/types/ui.d.ts

```diff
@@ -349,123 +349,6 @@ declare module 'ui' {
         }
     }
 
-    export namespace appDisplay {
```

Comment:
Why remove these? The typings are still correct, even if we are not using them anymore.

Reply:
Well most of them aren't valid anymore and since the class now exist on our side I felt it was pointless